README.md

```diff
@@ -6,6 +6,9 @@ This suite contains benchmarks drawn from three classes of tasks:
 - `Transformation.Text`: string-to-string transformation
 - `Split.Text`: text-to-table transformation
 - `Extraction.Text`: substring extraction from semi-structured text
+- `LastMile.Repair`: program repair for code which require very few edits to fix
```

Comment:
syntax (and some semantic) program repair for code that can be achieved with few edits